data/strings.yml

```diff
@@ -362,18 +362,29 @@ code_signing:
       in_progress: "Saving metadata, wait a sec..."
       success: "Keystore metadata successfully saved!"
 env_vars:
+  alert_make_it_protected_popup:
+    title: ‘Protected’ will be irreversible after save
+    details: If you choose to make this variable protected no one will be able to reveal the value, you can overwrite the value only by deleting the current one and creating a new one.
+    ok_button_title: Got it
   load_progress:
     in_progress: Loading, wait a sec...
+  get_secret_env_value_progress:
+    in_progress: Loading, wait a sec...
+    try_again_postfix: " Click here to try again."
   add_new: Add new
   replace_variables_label: "Replace variables in inputs?"
   expose_for_pr_label: "Expose for Pull Requests?"
   expose_for_pr_popover_content: You can decide which secrets should be exposed for / available in Pull Request builds.<br> Be careful, exposing a secret is a potential security risk.
-  delete_env_var: "Delete"
   invalid_env_var_key: "invalid Environment Variable key. Should not be empty, should only contain uppercase, lowercase letters, numbers, underscores, should not begin with a number"
+  invalid_secret_env_var_value: "invalid Secret Environment Variable value. Should not be empty"
```

Comment:
Move it below the next line, so value stuff comes after key stuff. Also, don't call out secret in the key name, nor in the error. Simply `invalid_env_var_value`, and the error message accordingly.